packages/docusaurus-init/templates/facebook/docs/doc3.md

```diff
@@ -0,0 +1,14 @@
+---
+id: doc3
+title: This is Document Number 3
+---
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. In ac euismod odio, eu consequat dui. Nullam molestie consectetur risus id imperdiet. Proin sodales ornare turpis, non mollis massa ultricies id. Nam at nibh scelerisque, feugiat ante non, dapibus tortor. Vivamus volutpat diam quis tellus elementum bibendum. Praesent semper gravida velit quis aliquam. Etiam in cursus neque. Nam lectus ligula, malesuada et mauris a, bibendum faucibus mi. Phasellus ut interdum felis. Phasellus in odio pulvinar, porttitor urna eget, fringilla lectus. Aliquam sollicitudin est eros. Mauris consectetur quam vitae mauris interdum hendrerit. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
Is there any extra information coming from this docs btw ? Like the other docs purpose is to demonstrate MDX, heading, etc. This lorem ipsum can be not super useful btw and just increase few seconds of first build.

I dont know why but I prefer having minimal init site, and make the user think "WOW its so fast !!!!"

Reply:
Not really. I don't mind cleaning up but in a future PR

Reply:
OOps btw just realized its been there since the beginning. You're just making a copy of it.

Maybe time to clean this up (?) wdyt